packages/vite/src/vite-node.ts

```
@@ -8,9 +8,12 @@
 import type { ModuleNode, Plugin as VitePlugin } from 'vite'
 import { getQuery } from 'ufo'
 import { normalizeViteManifest } from 'vue-bundle-renderer'
+import { resolve as resolveModule } from 'mlly'
 import { distDir } from './dirs'
 import type { ViteBuildContext } from './vite'
 import { isCSS } from './utils'
+import { createIsExternal } from './utils/external'
+import { transpile } from './utils/transpile'
```

ESLint

[error] 'transpile' is defined but never used.

(@typescript-eslint/no-unused-vars)